dbms/src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -139,9 +139,21 @@ namespace
                     config.keys(database_config, table_names);
 
                     /// Read table properties
-                    for (const String & table_name : table_names)
+                    for (const String & key_in_database_config : table_names)
```

Comment:
The name `keys_in_database_config` instead of `table_names` seems more consistent here.